air/tests/test_module/features/errors/last_error.rs

```diff
@@ -211,7 +211,7 @@ fn variable_names_shown_in_error() {
     let trace = trace_from_result(&result);
 
     assert_eq!(
-        trace[1],
+        trace.as_ref()[1],
```

Comment:
You implemented the `Index` trait for `ExecutionTrace`, why not just `trace[1]`?

Reply:
`ExecutionTrace` is to be indexable by `TracePos` only.

Reply:
Yep, I've changed on other places, but overlooked this one.

Reply:
Yep, it has been changed :)